setup.cfg

```
@@ -19,7 +19,7 @@ install_requires =
     astropy
     extinction
     lcdata>=1.1.1
-    lightgbm
+    lightgbm== 2.3.1
```

May I suggest to use less strict requirement
```suggestion
    lightgbm>=2.3.1,<3
```